tests/app/v2/template/test_get_template.py

```diff
@@ -0,0 +1,99 @@
+import pytest
+
+from flask import json
+
+from app import DATETIME_FORMAT
+from tests import create_authorization_header
+from tests.app.conftest import (
+    sample_template as create_sample_template
+)
+
+EMAIL_TYPE = 'email'
+SMS_TYPE = 'sms'
+LETTER_TYPE = 'letter'
+
+template_types = [EMAIL_TYPE, SMS_TYPE, LETTER_TYPE]
+valid_versions = [None, 1]
+
+
+@pytest.mark.parametrize("tmp_type", template_types)
+@pytest.mark.parametrize("version", valid_versions)
+def test_get_email_template_by_id_returns_200(client, notify_db, notify_db_session, sample_service, tmp_type, version):
+    template = create_sample_template(
```

Comment:
You can use the `create_template` fixture in `../tests/app/db.py` instead to avoid importing the `notify_db` and `notify_db_session`